src/components/checkbox/checkbox.scss

```diff
@@ -20,21 +20,19 @@
                 border-color: var(--lime-error-text-color);
             }
         }
-        &.mdc-ripple-upgraded {
-            .mdc-checkbox__native-control {
-                &:focus-visible {
-                    + .mdc-checkbox__background {
-                        &:after {
-                            content: '';
-                            display: block;
-                            position: absolute;
-                            top: -0.75rem;
-                            right: -0.75rem;
-                            bottom: -0.75rem;
-                            left: -0.75rem;
-                            border-radius: 50%;
-                            box-shadow: var(--shadow-depth-8-focused);
-                        }
+        .mdc-checkbox__native-control {
+            &:focus-visible {
+                + .mdc-checkbox__background {
+                    &:after {
+                        content: '';
+                        display: block;
+                        position: absolute;
+                        top: -0.75rem;
+                        right: -0.75rem;
+                        bottom: -0.75rem;
+                        left: -0.75rem;
+                        border-radius: 50%;
```

Comment:
```suggestion
                        inset: -0.25rem
                        border-radius: 0.25rem;
```

Reply:
The PR looks great. So happy we're removing these ripples finally! 🎉

We should also do this in a separate commit, where we decrease the amount of space waste for that visual effect of keyboard focus.

Reply:
I think that the "space waste" isn't because of the visual effect, but in order to provide a big enough size as a touch target.

Reply:
You're right about that Adrian. But we didn't change the touch area or anything. We just manipulated our custom `:after` element that is only used for visualization of the keyboard `focus`.